introduction/support.md

```diff
@@ -21,18 +21,14 @@ get the support you need, and we want to make sure our community remains a
 friendly place by ensuring we all follow the [Code of
 Conduct](/code-of-conduct/).
 
-## How to get help and support
+## Glossary of terms
+What does something mean? There may be a simple answer to that, in our [glossary of terms](https://www.qubes-os.org/doc/glossary/).
 
-First, let's see what kind of help you need.
+## Help and support
 
-### I would like to report a security vulnerability
+### Find a solution to a problem
 
-That sounds more like you helping us! Thanks! Please see [Reporting Security
-Issues in Qubes OS](/security/#reporting-security-issues-in-qubes-os).
-
-### I have a problem or a question
-
-No worries! Here's how we recommend proceeding:
+Qubes OS can be tough to learn and navigate. Here are some steps we suggest:
```

Comment:
It seems weird for us to say of our own OS that it's tough to learn and navigate. A reader might think, "If even *you* think it's tough to learn and navigate, then why did you *make* it that way?!"

Reply:
Empathy matters, and this is nicely empathetic. It takes a long time for software to become usable. Qubes does something no other software really does in a GUI. I think everyone understands that it's real humans who make it, and that we're not resource-rich enough to have something ultra-polished. This is good. :) Thank you for eliminating one exclamation point. There are currently very many—too many—in the docs. :)

Reply:
> Empathy matters, and this is nicely empathetic. It takes a long time for software to become usable. Qubes does something no other software really does in a GUI. I think everyone understands that it's real humans who make it, and that we're not resource-rich enough to have something ultra-polished. 

Oh, if only that were true! In my experience, it's quite the opposite.

Reply:
Also, I agree that empathy matters, which is why I tried to write this page in a more empathetic tone. It seems to me that many of the changes in this PR are moving from more empathetic to less empathetic. If that's not the case, I'd like to understand how it's not.

Reply:
That's good feedback! Intentions don't always come through as impact, and I'm not immune to that in my contributions.